src/widget/twitterEntrypoint.js

```diff
@@ -0,0 +1,50 @@
+import "chrome-extension-async";
+import "@webcomponents/custom-elements";
+import elementReady from "element-ready";
+
+import WidgetOrchestrator from "./widgetOrchestrator";
+
+import TwitterProfilePage from "../pages/twitterProfilePage";
+
+/**
+ * 
```

Comment:
Would be nice to populate this but not really needed. Either fill this in or remove the empty block ^^